### ISSUE TITLE
Input Slider heat temp disappears at lowest setting

### ISSUE THREAD

**Issue by sgauche** [home-assistant/home-assistant-polymer#147]
I have my min and max temp set to between 60F and 85F in my config, linked below.
https://github.com/sgauche/homeassistant-config/blob/master/customize.yaml#L77-L78

When I slide the slider all the way to the left, the blue drag bubble disappears.
https://i.gyazo.com/8892c944562ce1b5ae206fe512fecf53.gif

This also seems to be present on mobile.  Note that it doesn't happen for the high slider.
![image](https://cloud.githubusercontent.com/assets/8250717/19771081/b13d5576-9c2f-11e6-8131-491ab74bc023.png)

I'll tag @justweb1 because I know he has worked with the slider.

**Comment by justweb1** [comment]
I think it is an issue that has been fixed in paper-range-slider. we are waiting on a fix for another issue to upgrade it. I will check on the status.

**Comment by balloob** [comment]
Unable to reproduce in latest version.
